app/views/ShareListView/Header/index.tsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
+// @ts-ignore
+// eslint-disable-next-line import/extensions,import/no-unresolved
 import Header from './Header';
 
-const ShareListHeader = React.memo(({ searching, initSearch, cancelSearch, search, theme }) => {
-	const onSearchChangeText = text => {
+interface IShareListHeader {
+	searching: boolean;
+	initSearch?: () => void;
+	cancelSearch?: () => void;
+	search(text: string): void;
+	theme: string;
+}
```

Comment:
What do you think of creating a separate `interface.ts` file for an interface that you can share across the three files?